comet/scrapers/animetosho.py

```
@@ -0,0 +1,110 @@
+                torrents.append(
+                    {
+                        "title": title,
+                        "infoHash": info_hash,
+                        "fileIndex": None,
+                        "seeders": seeders,
+                        "size": size,
+                        "tracker": "AnimeTosho",
+                        "sources": extract_trackers_from_magnet(magnet),
+                    }
+                )
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)